apps/web/src/features/spaces/components/SpaceSettings/DeleteSpaceDialog.tsx

```diff
@@ -40,6 +42,7 @@ const DeleteSpaceDialog = ({ space, onClose }: { space: GetSpaceResponse | undef
     if (!space) return
 
     setError(undefined)
+    trackEvent({ ...SPACE_EVENTS.DELETE_SPACE })
```

Comment:
I would move this after the await call to make sure its only fired if the CGW request succeeded to keep it consistent with the other submit events.